cirq/sim/simulator.py

```diff
@@ -397,8 +397,12 @@ def __init__(self,
         self.measurements = measurements or collections.defaultdict(list)
 
     @abc.abstractmethod
-    def simulator_state(self) -> Any:
-        """Returns the simulator_state of the simulator after this step.
+    def _simulator_state(self) -> Any:
+        """Returns the simulator state of the simulator after this step.
+
+        This method starts with an underscore to indicate that it is private.
+        State that should be exposed should exposed on the child StepResult
```

Comment:
nit: This is a little convoluted. Maybe "Instead, use public state exposed by StepResult.".

Reply:
done